### DISCUSSION TITLE
[ERROR] Cannot start service: Host version "0.18.20" does not match binary version "0.19.4"

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#10676] [Help]
### Question

Not sure why this is happening but only when Vercel tries to build my website it gives this error:

[ERROR] Cannot start service: Host version "0.18.20" does not match binary version "0.19.4"
1 error
[vite:esbuild] The service was stopped: write EPIPE
file: /vercel/path0/src/pages/data.ts
 error   The service was stopped: write EPIPE
I removed package-lock.json and the whole node_modules folder to be sure but the error still shows up.

It's only happening with this [repo](https://github.com/jacobthesheep/torahcycle.info)

Also trying to create an issue on this repo sucks

**Comment by marcelooagte** [comment]
Check the global and local versions of your Vite.
In my case, the following occurred: First, run pnpm list vite for the local version:


“PS C:\MOA\Prog\projeto> pnpm list vite
Legend: production dependency, optional only, dev only

upload-ai-web@0.0.0 C:\MOA\Prog\projeto

devDependencies:
vite 4.4.9”

Then, run pnpm list -g create-vite for the global version: 

"PS C:\MOA\Prog\projeto> pnpm list -g create-vite
Legend: production dependency, optional only, dev only

C:\Users\SUPORTE\AppData\Local\pnpm\global\5

dependencies:
create-vite 4.4.1"

In the above case, I had to downgrade what was locally installed to a lower version to match both globally: 

pnpm install vite@4.4.1 --save-dev 


By doing this, my project started working again. I believe this might be your case as well.

**Comment by raulfdm** [comment]
overall insight: first try to upgrade vite-related dependencies, otherwise try to downgrade.

I had this issue with Astro and after a version bump, it was solved: https://github.com/raulfdm/three.js-journey/commit/25acdb28f9c39112d18325dd87c198400fbd962b
